frontend/app/src/components/import/GroupedImport.vue

```diff
@@ -50,55 +51,50 @@ import ShapeshiftImport from '@/components/import/ShapeshiftImport.vue';
 import UpholdImport from '@/components/import/UpholdImport.vue';
 import i18n from '@/i18n';
 
-const sources = () =>
-  computed(() => {
-    const sources = [
-      {
-        identifier: 'cointracking.info',
-        name: i18n.t('import_data.cointracking.name').toString(),
-        logo: require('@/assets/images/cointracking.svg'),
-        form: 'cointracking-import'
-      },
-      {
-        identifier: 'cryptocom',
-        name: i18n.t('import_data.cryptocom.name').toString(),
-        logo: require('@/assets/images/crypto_com.svg'),
-        form: 'crypto-com-import'
-      },
-      {
-        identifier: 'blockfi',
-        name: i18n.t('import_data.blockfi.name').toString(),
-        logo: require('@/assets/images/blockfi.svg'),
-        form: 'block-fi-import'
-      },
-      {
-        identifier: 'nexo',
-        name: i18n.t('import_data.nexo.name').toString(),
-        logo: require('@/assets/images/nexo.svg'),
-        form: 'nexo-import'
-      },
-      {
-        identifier: 'shapeshift-trades',
-        name: i18n.t('import_data.shapeshift.name').toString(),
-        logo: require('@/assets/images/shapeshift.svg'),
-        form: 'shapeshift-import'
-      },
-      {
-        identifier: 'uphold',
-        name: i18n.t('import_data.uphold.name').toString(),
-        logo: require('@/assets/images/uphold.svg'),
-        form: 'uphold-import'
-      },
-      {
-        identifier: 'bisq',
-        name: i18n.t('import_data.bisq.name'),
-        logo: require('@/assets/images/bisq.svg'),
-        form: 'bisq-import'
-      }
-    ];
-
-    return sources;
-  });
+const sources = [
```

Comment:
This change is ok for now but we might need to revert it in the future
after updating to the latest vue-i18n https://vue-i18n.intlify.dev/guide/advanced/composition.html

We don't have an issue for it, but at some point, we should at least upgrade to the [bridge](https://vue-i18n.intlify.dev/guide/migration/vue2.html#what-is-vue-i18n-bridge) and start preparing, probably at some point after merging the vite PR